shared/teams/add-members-wizard/add-contacts.native.tsx

```diff
@@ -43,9 +43,10 @@ const AddContacts = () => {
         if (r?.length) {
           dispatch(
             TeamsGen.createAddMembersWizardPushMembers({
-              members: r.map(m => ({
-                assertion: m.foundUser ? m.username : m.assertion,
-                note: m.foundUser ? formatPhoneNumber(m.input) : undefined,
+              members: r.map<Types.AddingMember>(m => ({
```

Comment:
type annotation needed?

Reply:
probably not... I think I got annoyed by lack of error when i had a typo in the object literal but this probably doesn't help

Reply:
there should be an error, it should complain that it doesn't match the action payload type. the spread might be causing it not to error.